frontend/js/components/Repeater.vue

```diff
@@ -77,7 +77,22 @@
       blocks: {
         get () {
           if (this.savedBlocks.hasOwnProperty(this.name)) {
-            return this.savedBlocks[this.name] || []
+            const blocks = this.savedBlocks[this.name]
+            if (blocks && blocks.length > 0) {
+              const repeaterName = this.name.replace(/(blocks-\d+_)/, '')
+              const titleFieldToMap = this.$store.state.repeaters.availableRepeaters[repeaterName] && this.$store.state.repeaters.availableRepeaters[repeaterName].titleField ? this.$store.state.repeaters.availableRepeaters[repeaterName].titleField : null
+              if (titleFieldToMap) {
+                return blocks.map(block => {
+                  const titleField = this.$store.state.form.fields.find(field => field.name === `blocks[${block.id}][${titleFieldToMap}]`)
+                  if (titleField && titleField.value) block.title = titleField.value
```

Comment:
There is an issue with `titleField.value` if we have a form with multi languages enabled.
The returned value is an object with value for each language. We need to detect if the form have multi languages and get the value for the current language.

<img width="756" alt="screenshot 2019-01-17 at 22 25 02" src="https://user-images.githubusercontent.com/11910547/51353049-b46aae00-1aaf-11e9-9073-001dbca030b4.png">